packages/@ember/-internals/runtime/lib/system/array_proxy.ts

```diff
@@ -226,7 +226,7 @@ class ArrayProxy<T, C extends EmberArray<T> = EmberArray<T>>
     let content = get(this, 'content');
     assert('[BUG] Called objectAtContent without content', content);
     assert('Mutating a non-mutable array is not allowed', isMutable(content));
-    content.replace(idx, amt, objects);
+    replace(content, idx, amt, objects);
```

Comment:
`content.replace` would only work on EmberArray, but the utility method `replace` will work on either.